src/libpgagroal/management.c

```diff
@@ -1113,10 +1114,12 @@ pgagroal_management_read_isalive(SSL* ssl, int socket, int* status, char output_
 int
 pgagroal_management_write_isalive(int socket, bool gracefully)
 {
-   char buf[4];
+   char buf[MANAGEMENT_HEADER_SIZE];
 
    memset(&buf, 0, sizeof(buf));
 
+   write_header_info(buf, 1, "pgagroal ");
```

Comment:
Second argument of `write_header_info` is a `bool`, so use `true` or `false`.

Reply:
will do